spec/lib/diaspora/fetcher/public_spec.rb

```diff
@@ -216,7 +216,7 @@
 
     describe '#check_existing' do
       it 'returns false if a post with the same guid exists' do
-        post = {'guid' => FactoryGirl.create(:status_message).guid}
+        post = {'guid' => FactoryBot.create(:status_message).guid}
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.